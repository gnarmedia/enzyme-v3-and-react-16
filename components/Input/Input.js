/**
 * @file Input component
 */

import React from 'react'

const Input = (props) => (
  <div className="field">
    <div className="control">
      <input className="input is-large" {...props} />
    </div>
  </div>
)

Input.defaultProps = {
  placeholder: null,
  type: 'text'
}

export default Input