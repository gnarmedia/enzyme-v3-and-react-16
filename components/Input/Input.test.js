import React from 'react'
import { shallow } from 'enzyme'

import Input from './Input'

describe('Input', () => {
  it('should render the A component', () => {
    const wrapper = shallow(<Input value="some text" />)

    expect(wrapper).toMatchSnapshot()
  })
})