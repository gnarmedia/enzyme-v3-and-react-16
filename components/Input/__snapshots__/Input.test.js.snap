// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render the A component 1`] = `
<div
  className="field"
>
  <div
    className="control"
  >
    <input
      className="input is-large"
      placeholder={null}
      type="text"
      value="some text"
    />
  </div>
</div>
`;
